jpos/src/dist/deploy/00_logger.xml

```diff
@@ -3,6 +3,31 @@
 <logger name="Q2" class="org.jpos.q2.qbean.LoggerAdaptor">
   <property name="redirect" value="stdout, stderr" />
   <log-listener class="org.jpos.util.SimpleLogListener" />
+  <log-listener class="org.jpos.util.BufferedLogListener">
+    <property name="max-size" value="100" />
+    <property name="name" value="logger.Q2.buffered" />
+  </log-listener>
+
+  <log-listener class="org.jpos.util.KafkaLogListener">
```

Comment:
IMO, this logger file should not add Kafka listener - perhaps provide this as an example elsewhere?